### PR TITLE
Add contributing guides for blog authorship

### PR DESCRIPTION
**Pull request by BenWilson2**
Add the blog contribution guidelines and contributing guides for MLflow Ambassadors

### CODE REVIEW

File: DEVELOPMENT_GUIDE.md
Comment (harupy) [inline DEVELOPMENT_GUIDE.md:82]: Can we just add a link to the official installation guide?

File: DEVELOPMENT_GUIDE.md
Comment (harupy) [inline DEVELOPMENT_GUIDE.md:93]: The same comment for yarn.

https://github.com/mlflow/mlflow-website/pull/48/files#r1545880010

File: DEVELOPMENT_GUIDE.md
Comment (harupy) [inline DEVELOPMENT_GUIDE.md:5]:
```suggestion
The core of the MLflow website is powered by [Docusaurus](https://docusaurus.io/).
```

I don't think we need to mention what `Docusaurus` is.

File: DEVELOPMENT_GUIDE.md
Comment (harupy) [inline DEVELOPMENT_GUIDE.md:146]:
```suggestion
...
```

File: DEVELOPMENT_GUIDE.md
Comment (harupy) [inline DEVELOPMENT_GUIDE.md:202]: This directory should be gitignored.

File: DEVELOPMENT_GUIDE.md
Comment (harupy) [inline DEVELOPMENT_GUIDE.md:9]: Not sure if we need to mention this because the only way for an external contributor to file a PR is to file a PR from their fork.

File: DEVELOPMENT_GUIDE.md
Comment (harupy) [reply DEVELOPMENT_GUIDE.md:146]: I'd remove the output section. It doesn't seem important.

File: CONTRIBUTING.md
Comment (harupy) [inline CONTRIBUTING.md:41]: I think we can release a new blog anytime we want. Can we adjust the wording here?

File: DEVELOPMENT_GUIDE.md
Comment (BenWilson2) [reply DEVELOPMENT_GUIDE.md:82]: Great point. Changed all of this and any non-our-code examples to point to source package guides :)

File: DEVELOPMENT_GUIDE.md
Comment (BenWilson2) [reply DEVELOPMENT_GUIDE.md:146]: yep deleted all of that and just mentioned that a local url can be used for active dev

File: CONTRIBUTING.md
Comment (BenWilson2) [reply CONTRIBUTING.md:41]: We don't want to set that precedent :) Otherwise, you and I will be pinged daily about "could you release my blog?". I want external blogs to match with our release cadence to limit the amount of questions and pings that we get. We're going to be having ~20-30 blogs / quarter and I don't want us to do more than 2-3 releases per quarter :)

Comment (AbeOmor) [reply CONTRIBUTING.md:41]: Can we add a link the the Mlflow Project here.. So people know what the next releases are..

I agree with @harupy we will probably need a process in the future that allow folks to publish after it is approved, rather than waiting for the MLflow version release

File: CONTRIBUTING.md
Comment (AbeOmor) [inline CONTRIBUTING.md:47]: Pages? on what Google Docs or Word?

It is hard to know. Maybe we should clarified.... 3-8 pages in a google docs.

File: CONTRIBUTING.md
Comment (AbeOmor) [inline CONTRIBUTING.md:74]: 🤣
> (We don't have a headshot or LinkedIn profile for GPT-4, DBRX, or Mixtral, after all)

File: CONTRIBUTING.md
Comment (AbeOmor) [inline CONTRIBUTING.md:140]: Do we want to add somewhere, their image is sourced from LinkedIN profile

**Review by AbeOmor — APPROVED** [review]
LGTM, with minor nits

File: CONTRIBUTING.md
Comment (BenWilson2) [reply CONTRIBUTING.md:41]: They won't be releasing. We would have to release. And we're not taking on the maintenance burden of releasing 5 blogs / week.

File: CONTRIBUTING.md
Comment (BenWilson2) [reply CONTRIBUTING.md:47]: "rendered". I'll make it more clear.

File: CONTRIBUTING.md
Comment (BenWilson2) [reply CONTRIBUTING.md:140]: Good call! Added

**Pull request merged**
